src/nft/components/explore/Banner.tsx

```diff
@@ -0,0 +1,185 @@
+import clsx from 'clsx'
+import { Box } from 'nft/components/Box'
+import { Center, Column, Row } from 'nft/components/Flex'
+import { VerifiedIcon } from 'nft/components/icons'
+import { bodySmall, buttonMedium, header1, section } from 'nft/css/common.css'
+import { vars } from 'nft/css/sprinkles.css'
+import { fetchTrendingCollections } from 'nft/queries'
+import { TimePeriod, TrendingCollection } from 'nft/types'
+import { formatEthPrice } from 'nft/utils/currency'
+import { putCommas } from 'nft/utils/putCommas'
+import { formatChange, toSignificant } from 'nft/utils/toSignificant'
+import { ReactNode, useEffect, useState } from 'react'
+import { useQuery } from 'react-query'
+import { Link } from 'react-router-dom'
+
+import * as styles from './Banner.css'
+
+const Banner = () => {
+  /* Sets initially displayed collection to random number between 0 and 4  */
+  const [current, setCurrent] = useState(Math.floor(Math.random() * 5))
+  const [hovered, setHover] = useState(false)
+  const { data: collections } = useQuery(
+    ['trendingCollections'],
+    () => {
+      return fetchTrendingCollections({ volumeType: 'eth', timePeriod: TimePeriod.OneDay, size: 5 })
+    },
+    {
+      refetchOnReconnect: false,
+      refetchOnWindowFocus: false,
+      refetchOnMount: false,
+    }
+  )
+
+  useEffect(() => {
+    /* Rotate through Top 5 Collections on 15 second interval */
+    const interval = setInterval(async () => {
+      if (collections && !hovered && !stale) {
```

Comment:
I think it makes sense to add a guard statement with hovered and stale:
```
if (hovered || stale) return
if (collections) {
  ...
}
```

wdyt?

Reply:
agree - cleaner that way